guides/common/modules/proc_acd_installation.adoc

```diff
@@ -45,6 +45,41 @@ endif::[]
 # {foreman-maintain} service restart
 ----
 
+[id="{context}_installation_{SmartProxyServer}"]
+= Installing ACD on {SmartProxyServer}
+
+You need to install two packages to use ACD on your {SmartProxy}.
+
+.Procedure
+ifdef::foreman-el,foreman-deb,katello,satellite[]
+. Create a yum repository `/etc/yum.repos.d/foreman-plugins.repo` on your {SmartProxyServer}:
++
+[source,bash,subs="+quotes,attributes"]
+----
+[foreman-plugins]
+name=Foreman plugins
+baseurl=https://yum.theforeman.org/plugins/nightly/el7/x86_64/
+enabled=1
+gpgcheck=0
+----
+endif::[]
+. Install the required packages on your {SmartProxyServer}:
++
+[source,bash]
+----
+# yum install tfm-rubygem-smart_proxy_acd tfm-rubygem-smart_proxy_acd_core
+----
+. Restart the {SmartProxy} services:
++
+[source,bash]
+----
+# systemctl restart foreman-proxy
+----
+. Refresh {SmartProxy} features on {ProjectServer}:
++
+Navigate to *Infrastructure* > *Smart Proxies* and select your {SmartProxyServer}.
+Click the *Refresh* button in the *Actions* drop down menu.
```

Comment:
```suggestion
From the *Actions* list, click *Refresh*.
```
The most interesting part of the IBM style guide is "Graphical User Interface Elements" - I'm not sure if you have a copy yet, but it is great.

Reply:
Fixed.

And yes, I finally got my own copy.